src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -83,19 +83,51 @@ function WorkspaceInvitePage(props) {
     const excludedUsers = useMemo(() => PolicyUtils.getIneligibleInvitees(props.policyMembers, props.personalDetails), [props.policyMembers, props.personalDetails]);
 
     useEffect(() => {
-        const inviteOptions = OptionsListUtils.getMemberInviteOptions(props.personalDetails, props.betas, searchTerm, excludedUsers);
-
-        // Update selectedOptions with the latest personalDetails and policyMembers information
-        const detailsMap = {};
-        _.forEach(inviteOptions.personalDetails, (detail) => (detailsMap[detail.login] = OptionsListUtils.formatMemberForList(detail)));
-        const newSelectedOptions = [];
-        _.forEach(selectedOptions, (option) => {
-            newSelectedOptions.push(_.has(detailsMap, option.login) ? {...detailsMap[option.login], isSelected: true} : option);
+        let emails = searchTerm.replace(/\s,\s/g, ',').split(',');
+        emails = _.filter(
+            _.map(emails, (word) => word.trim()),
+            (email) => email !== '',
+        );
+
+        const newUsersToInviteDict = {};
+        const newPersonalDetailsDict = {};
+        const newSelectedOptionsDict = {};
+
+        _.forEach(emails, (email) => {
```

Comment:
Sorry this might sound nitpick, but it seems like we mostly standardized on `each()` (`forEach()` is an alias). Can we replace those with `.each()` for consistency please?

```suggestion
        _.each(emails, (email) => {
```